be/src/storage/metadata_util.cpp

```diff
@@ -343,6 +343,27 @@ Status convert_t_schema_to_pb_schema(const TTabletSchema& tablet_schema, uint32_
                 properties_map.emplace(INDEX_PROPERTIES, index.index_properties);
                 std::string str = to_json(properties_map);
                 index_pb->set_index_properties(str);
+            } else if (index.index_type == TIndexType::type::VECTOR) {
+                RETURN_IF(index.columns.size() != 1,
+                          Status::Cancelled("VECTOR index " + index.index_name +
+                                            " do not support to build with more than one column"));
+                index_pb->set_index_type(IndexType::VECTOR);
+
+                const auto& index_col_name = index.columns[0];
+                const auto& mit = column_map.find(boost::to_lower_copy(index_col_name));
```

Comment:
Why do you call `to_lower_copy` here? I think this should be done in the frontend. And in the storage engine, it should respect the name from frontends.

Reply:
> Why do you call `to_lower_copy` here? I think this should be done in the frontend. And in the storage engine, it should respect the name from frontends.

history reason, every index need call to_lower_copy

Reply:
So do I need to ensure the respect from frontends in this pr, or refactor it in a separate pr in the future?